behavior/get_spatial_trace_from_behavior.m

```diff
@@ -0,0 +1,62 @@
+function spatial_trace = get_spatial_trace_from_behavior(sample_image,video)   
+%function spatial_trace = get_spatial_trace_from_behavior(sample_image,video)
+%
+%Extract X and Y coordinates of the mouse from the behavior video
+%
+%   sample_image : A snapshot the behavior setup for use in the maze region
+%   extraction step.
+%
+%   video : [Height x Width x num_frames] array
+%
+%   spatial_trace : [num_frames x 2] matrix where the 1st column is the X-coordinate and
+%   the 2nd column is the Y-coordinate of the mouse. 
+
+    
+    num_frames = size(video,3);
+    centroids = zeros(num_frames,2); 
+    
+    %anything above thresh_whiteness is considered white
+    thresh_whiteness = 100;
+    
+    %anything below thresh_blackness is considered black
+    thresh_blackness = 20;
+    c_old = [0 0];
+    
+    %extract roughly the maze from the sample image (based on whiteness)
+    if size(sample_image,3)>1
+        maze_image = rgb2graysample_image; %convert to grayscale
+    else
+        maze_image = sample_image;
+    end
+    maze_image = medfilt2(maze_image,[20,20]); %blur
```

Comment:
I used it to get rid of details in the scene, which I still see as a kind of blurring. I changed the comment tho.